sensor/common/component.go

```diff
@@ -6,11 +6,20 @@ import (
 	"google.golang.org/grpc"
 )
 
+// SensorComponentEvent represents events about which sensor components can be notified
+type SensorComponentEvent string
+
+const (
+	// SensorComponentEventCentralReachable denotes that Sensor-Central connection is up
+	SensorComponentEventCentralReachable SensorComponentEvent = "central-reachable"
```

Comment:
Optional nit: Though this is a very expressive var name, it is very long.
I struggle to come up with some shorter ones as well, but how about something like `CentralReached` or `CentralContacted`?